docs/specs/Mountain State Adventure Destination/SPEC-13-template-lake/data-model.md

```
@@ -0,0 +1,658 @@
+  /** Array of target fish species for this spot */
+  species: z.array(z.string().min(1)).min(1),
+
+  /** Access method (e.g., "Boat only", "Shore accessible", "Kayak launch nearby") */
+  access: z.string(),
+});
+
+export type FishingSpot = z.infer<typeof FishingSpotSchema>;
+```
+
+**Usage Example**:
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)